packages/fast-components-react-msft/src/pivot/pivot.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Pivot",
+    "description": "A pivot component's schema definition.",
+    "type": "object",
+    "id": "pivot",
+    "properties": {
+        "activeId": {
+            "title": "Active pivot ID",
+            "type": "string"
+        },
+        "label": {
+            "title": "HTML aria-label attribute",
+            "type": "string"
+        },
+        "orientation": {
+            "title": "HTML aria-orientation attribute",
+            "type": "string",
+            "enum": [
+                "horizontal",
+                "vertical"
+            ]
+        },
+        "items": {
+            "title": "Items",
+            "type": "array",
+            "items": {
+                "title": "Item",
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "title": "HTML id attribute",
+                        "type": "string"
+                    }
+                },
+                "required": [

```

Comment:
This required is referencing properties not found in items?

Reply:
Items is of type TabsItem which consists of:
```
export interface TabsItem {
    tab: (className: string) => React.ReactNode;
    content: (className: string) => React.ReactNode;
    id: string;
}
```